File: periodo-4/desafios/lista02/1041.py
x, y = input().split(' ')
x, y = float(x), float(y)

o = 0.0
c = 'Origem'

if x != o or y != o:
	if x > o and y > o:
		c = 'Q1'
	elif x < o and y > o:
		c = 'Q2'
	elif x < o and y < o:
		c = 'Q3'
	elif x > o and y < o:
		c = 'Q4'
	elif x == o:
		c = 'Eixo Y'
	else:
		c = 'Eixo X'

print(c)

File: periodo-4/desafios/lista05/1041.c
#include <stdio.h>

int main ()
{
	int c1, c2, c3, c4, c5, c6, c7;
	float point_x, point_y;
	scanf("%f%f", &point_x, &point_y);

	if (point_x == point_y && point_y == 0.0) // ponto da origem
		printf("Origem\n");
	else if (point_x > 0.0 && point_y > 0.0) // 1° quadrante
		printf("Q1\n");
	else if (point_x < 0.0 && point_y > 0.0) // 2° quadrante
		printf("Q2\n");
	else if (point_x < 0.0 && point_y < 0.0) // 3° quadrante
		printf("Q3\n");
	else if (point_x > 0.0 && point_y < 0.0) // 4° quadrante
		printf("Q4\n");
	else if (point_x == 0.0 && point_y != 0.0) // eixo point_y
		printf("Eixo Y\n");
	else // eixo x
		printf("Eixo X\n");

	return 0;
}